source/crud/tests/v2/aggregate-out-readConcern.yml

```diff
@@ -0,0 +1,155 @@
+data:
+    - {_id: 1, x: 11}
+    - {_id: 2, x: 22}
+    - {_id: 3, x: 33}
+
+minServerVersion: '4.1'
+
+collection_name: &collection_name 'testaggcollection'
+
+tests:
+  -
+    description: "readConcern majority with out stage"
+    operations:
+      -
+        object: collection
+        name: aggregate
+        collectionOptions:
+          readConcern:
+            level: "majority"
+        arguments:
+          pipeline:
+            - $sort: {x : 1}
+            - $match:
+                _id: {$gt: 1}
+            - $out: &output_collection "other_test_collection"
+    expectations:
+      -
+        command_started_event:
+          command:
+            aggregate: *collection_name
+            pipeline:
+              - $sort: { x: 1 }
+              - $match: { _id: { $gt: 1 } }
+              - $out: *output_collection
+            readConcern: { level: "majority" }
+    outcome:
+      collection:
+        name: *output_collection
+        data:
+          - {_id: 2, x: 22}
+          - {_id: 3, x: 33}
+  -
+    description: "readConcern local with out stage"
+    operations:
+      -
+        object: collection
+        name: aggregate
+        collectionOptions:
+          readConcern:
+            level: "local"
+        arguments:
+          pipeline:
+            - $sort: {x : 1}
+            - $match:
+                _id: {$gt: 1}
+            - $out: &output_collection "other_test_collection"
```

Comment:
It shouldn't be necessary to redefine this anchor. I think you can just use `*output_collection` here.